resources/public/api/conf/1.0/application.raml

```diff
@@ -413,13 +427,15 @@ traits:
     get:
       displayName: Get all Movement Departures
       description: |
-        You may opt to filter this list of Departures so that you only receive Movement Departures that have been updated since a specified date and time.
+        Retrieve all movement departures sent to the Customs Office of Departure within 31 days of the release of a transit movement at its final destination.
```

Comment:
This isn't correct. This is just getting all departure declarations, regardless of whether they've been released - you can get a movement that has been declared as soon as it has been declared.

Also, the 31 days thing is as above.

Reply:
Will address.